api/client.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/infrahq/infra/uid"
 )
 
-var apiVersion = "0.18.1"
```

Comment:
Just validating, this is the correct way to update the CLI's API version?

Reply:
Yes but 0.20.1 is not a valid version